src/ucp/wireup/select.c

```diff
@@ -985,12 +1044,14 @@ ucs_status_t ucp_wireup_select_lanes(ucp_ep_h ep, const ucp_ep_params_t *params,
         }
     }
 
-    /* Sort RMA and AMO lanes according to score */
+    /* Sort RNDV, RMA and AMO lanes according to score */
     ucs_qsort_r(key->rma_lanes, UCP_MAX_LANES, sizeof(ucp_lane_index_t),
                 ucp_wireup_compare_lane_rma_score, lane_descs);
     ucs_qsort_r(key->amo_lanes, UCP_MAX_LANES, sizeof(ucp_lane_index_t),
                 ucp_wireup_compare_lane_amo_score, lane_descs);
 
+    /* Do not sort rndv lanes to save original sequence */
```

Comment:
it seems to contradict the comment '/* Sort RNDV, RMA and AMO lanes according to score */'

Reply:
comment is updated